zaius/export/api.py

```diff
@@ -140,20 +142,46 @@ def _s3_download(self, s3_url, local_path):
             aws_secret_access_key=self.auth["aws_secret_access_key"],
         )
         kwargs = {"Bucket": bucket, "Prefix": prefix}
+        keys = []
         while True:
             objs = s3_client.list_objects_v2(**kwargs)
-
-            for obj in objs["Contents"]:
-                _, fname = os.path.split(obj["Key"])
-                self.log.info("downloading {}".format(obj["Key"]))
-                s3_client.download_file(
-                    bucket, obj["Key"], os.path.join(local_path, fname)
-                )
-
+            keys.extend([obj["Key"] for obj in objs["Contents"]])
```

Comment:
```suggestion
            keys.extend(obj["Key"] for obj in objs["Contents"])
```
Further to the use of extend, you can also use the comprehension directly as the argument here as it's iterable, which then avoids materialising it into a new list before passing it.